src/patternfly/components/Card/card.scss

```diff
@@ -323,6 +323,12 @@
   overflow-wrap: var(--#{$card}__title-text--OverflowWrap);
 }
 
+.#{$card}__subtitle {
+  padding-block-start: var(--pf-t--global--spacer--sm);
+  font-size: var(--pf-t--global--font--size--body--sm);
+  color: var(--pf-t--global--text--color--subtle);
```

Comment:
Can you create component vars for these at the top of the scss file, then use those vars here?

```css
.#{$card} {
  --#{$card}__subtitle--PaddingBlockStart: var(--pf-t--global--spacer--sm);
  --#{$card}__subtitle--FontSize: var(--pf-t--global--font--size--body--sm);
  --#{$card}__subtitle--Color: var(--pf-t--global--text--color--subtle);
}

.#{$card}__subtitle {
  padding-block-start: var(--#{$card}__subtitle--PaddingBlockStart);
  font-size: var(--#{$card}__subtitle--FontSize);
  color: var(--#{$card}__subtitle--Color);
}
```